job_spec/SRG_GSLC.yml

```diff
@@ -24,20 +24,28 @@ SRG_GSLC:
   cost_profiles:
     DEFAULT:
       cost: 1.0
+  compute_environment:
+    name: SrgGslc
+    instance_types:
+      - g6.2xlarge
+    # Image ID for: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
+    ami_id: ami-0ca2af66da8e56876
   tasks:
     - name: ''
       image: ghcr.io/asfhyp3/hyp3-srg
       command:
         - ++process
         - back_projection
+        - --gpu
         - --bucket
         - '!Ref Bucket'
         - --bucket-prefix
         - Ref::bucket_prefix
         - Ref::granules
       timeout: 10800
       vcpu: 1
-      memory: 31500
+      gpu: 1
+      memory: 30500
```

Comment:
Just want to confirm that it was intentional to decrease the memory allocation slightly?